extensions/data-workspace/src/test/dialogs/openExistingDialog.test.ts

```diff
@@ -36,7 +36,7 @@ suite('Open Existing Dialog', function (): void {
 		should.equal(await dialog.validate(), true, 'Validation pass because project file exists');
 	});
 
-	test('Should validate workspace file exists', async function (): Promise<void> {
+	test.skip('Should validate workspace file exists', async function (): Promise<void> {
```

Comment:
Instead of skipping, what about changing `dialog.validate()` to throw/not throw instead of return true/false.  This'll let us capture the actual error instead of just failing on a boolean.

We can add the skip once we have the underlying issue captured and if it's still not immediately obvious how to fix.

Reply:
Clicked approve too soon, I agree with Benjin, if you simply skip the test, it will probably very hard for you to reproduce the error, it would be easier if you add more logging and wait for a repro and then investigate.

Reply:
@SakshiS-harma did you get a chance to chat with @alanrenmsft about this one?

@alanrenmsft I think this was skipped  for now to ensure product builds or DRI arent blocked during holidays if we arent available.